Workbooks/SapMonitor2.0/AIOpsInsights/SAP AMS.workbook

```diff
@@ -31,7 +31,7 @@
             "name": "workspace",
             "type": 5,
             "isRequired": true,
-            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"/subscriptions/{amsResource:subscriptionid}/resourceGroups/{amsResource:resourcegroup}/providers/Microsoft.Workloads/monitors/{amsResource:name}\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2022-11-01-preview\"}],\"batchDisabled\":true,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.properties\",\"columns\":[{\"path\":\"$.logAnalyticsWorkspaceArmId\",\"columnid\":\"id\",\"columnType\":\"string\"}]}}]}",
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"/subscriptions/{amsResource:subscriptionid}/resourceGroups/{amsResource:resourcegroup}/providers/Microsoft.Workloads/monitors/{amsResource:name}\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2021-12-01-preview\"}],\"batchDisabled\":true,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.properties\",\"columns\":[{\"path\":\"$.logAnalyticsWorkspaceArmId\",\"columnid\":\"id\",\"columnType\":\"string\"}]}}]}",
```

Comment:
please use "2023-04-01" API version for making calls to Microsoft.Workloads. It is our GA release version. Please change this in all places